src/script/lua_api/l_mainmenu.cpp

```diff
@@ -1039,25 +1039,17 @@ int ModApiMainMenu::l_get_package_list(lua_State *L)
 		lua_pushstring(L, package.shortDesc.c_str());
 		lua_settable  (L, top_lvl2);
 
-		lua_pushstring(L, "url");
-		lua_pushstring(L, package.url.c_str());
-		lua_settable  (L, top_lvl2);
-
 		lua_pushstring (L, "release");
 		lua_pushinteger(L, package.release);
 		lua_settable   (L, top_lvl2);
 
-		lua_pushstring(L, "screenshots");
-		lua_newtable(L);
-		{
-			int top_screenshots = lua_gettop(L);
-			for (size_t i = 0; i < package.screenshots.size(); ++i) {
-				lua_pushnumber(L, i + 1);
-				lua_pushstring(L, package.screenshots[i].c_str());
-				lua_settable(L, top_screenshots);
-			}
-		}
-		lua_settable(L, top_lvl2);
+		lua_pushstring(L, "thumbnail");
+		lua_pushstring(L, package.thumbnail.c_str());
+		lua_settable  (L, top_lvl2);
+
+		lua_pushstring(L, "url");
+		lua_pushstring(L, (url + "/packages/" + package.author + "/" + package.name + "/download/").c_str());
```

Comment:
Line is too long. Please move this URL building sequence to a helper function in `struct Package`